pkgs/applications/video/olive-editor/default.nix

```diff
@@ -0,0 +1,41 @@
+{ stdenv, fetchFromGitHub, pkgconfig, which, qt5, frei0r, opencolorio, ffmpeg-full  }:
+
+stdenv.mkDerivation
+{
+  pname = "olive-editor";
+
+  version = "2019-03-18";
```

Comment:
> https://nixos.org/nixpkgs/manual/#sec-package-naming

> If a package is not a release but a commit from a repository, then the version part of the name must be the date of that (fetched) commit. The date must be in "YYYY-MM-DD" format. Also append "unstable" to the name - e.g., "pkgname-unstable-2014-09-23". 

So you just need the `unstable` as a part of `version`. Note the docs don't say that because we haven't updated the docs.